.github/workflows/test.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - '!*'
     branches:
       - '*'
-  pull_request:
+  pull_request_target:
```

Comment:
According to this [GitHub article](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/) this could be potentially unsafe and it's encouraged to go with the approach of using the `workflow-run` trigger.